xml/depl_nodes.xml

```diff
@@ -4854,42 +4850,40 @@ openstack router add subnet <replaceable>router2</replaceable> priv-net-sub</scr
   <variablelist>
    <varlistentry>
     <term>
-     Interval used for CPU/disk/network/other meter updates (in seconds)
+     Interval used for &ostack; Compute meter updates (in seconds)
     </term>
     <listitem>
      <para>
-      Specify an interval in seconds after which &o_meter; performs an update
-      of the specified meter.
+      Interval used for &ostack; image meter updates (in seconds)
```

Comment:
I'm not sure if this will render correctly. We have three setting "titles" here and want a common description for them.

Reply:
Pinged you about this one :)